Apps/MAUI.Demo/ModulePages/CrashesContentPage.xaml.cs

```diff
@@ -0,0 +1,276 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using MAUI.Demo.Views;
+using Microsoft.AppCenter;
+using Microsoft.AppCenter.Crashes;
+using System.Linq;
+using System.Threading.Tasks;
+//using MAUI.Demo.UtilClassLibrary;
+
+namespace MAUI.Demo;
+
+public class NonSerializableException : Exception
+{
+}
+
+public partial class CrashesContentPage
+{
+    public const string TextAttachmentKey = "TEXT_ATTACHMENT";
+
+    public const string FileAttachmentKey = "FILE_ATTACHMENT";
+
+    List<Property> Properties;
+
+    public CrashesContentPage()
+    {
+        InitializeComponent();
+        Properties = new List<Property>();
+        NumPropertiesLabel.Text = Properties.Count.ToString();
+        if (DeviceInfo.Current.Platform == DevicePlatform.iOS)
+        {
+            //Icon = "socket.png";
+        }
+    }
+
+    protected override async void OnAppearing()
+    {
+        base.OnAppearing();
+        RefreshEnabled();
+        var hasLowMemoryWarning = await Crashes.HasReceivedMemoryWarningInLastSessionAsync();
+        MemoryWarningLabel.Text = hasLowMemoryWarning ? "Yes" : "No";
```

Comment:
Should we move such things into resources?

Reply:
You mean all UI strings in app? May be, but again, Forms app doesn't have it in resources. Probably can skip as we need to merge it fast.